ferrocene/tools/generate-tarball/src/signatures.rs

```diff
@@ -78,7 +79,10 @@ fn collect_files(
             package.files.push(PackageFile {
                 path: relative_path.into(),
                 sha256: hash_file(&entry)?,
+                #[cfg(not(windows))]
                 posix_mode: entry.metadata()?.mode(),
+                #[cfg(windows)]
+                posix_mode: 0, // TODO: Not used! Should we cfg it in criticaltrust?
```

Comment:
Let's not cfg in criticaltrust: a Linux criticaltrust should be able to understand Windows criticaltrust files (otherwise the release process can't parse them).